lib/shipit.rb

```diff
@@ -103,8 +103,9 @@ def host
     secrets.host.presence
   end
 
-  def github_team
-    @github_team ||= github_oauth_credentials['team'] && Team.find_or_create_by_handle(github_oauth_credentials['team'])
+  def github_teams
+    teams = (Array(github_oauth_credentials['team']) + Array(github_oauth_credentials['teams'])).sort.uniq
+    @github_teams ||= teams.any? && teams.map { |t| Team.find_or_create_by_handle(t) }
```

Comment:
Yeah, let's return an empty array instead of false, combined with the other comment:

```ruby
>> Shipit::Team.where(id: []).exists?
  Shipit::Team Exists (0.4ms)  SELECT  1 AS one FROM `teams` WHERE 1=0 LIMIT 1
```

So that sounds good. (Yes we could avoid the query, but I prefer simpler code)